### DISCUSSION TITLE
Why is 0x62AA0000 specified as the base address in the Windows version build?

### DISCUSSION THREAD

**Discussion by KiyonoriKami** [nigels-com/glew#423] [Q&A]
The following warning appears at build time. Why do you specify a base address?

1>------ Build started: Project: glew_shared, Configuration: Release x64 ------
1>glew.c
1>   Creating library ..\..\lib\Release\x64\glew32.lib and object ..\..\lib\Release\x64\glew32.exp
1>LINK : warning LNK4281: undesirable base address 0x62AA0000 for x64 image; set base address above 4GB for best ASLR optimization

**Comment by nigels-com** [comment]
It probably makes sense to disable this for 64-bit.
Or bump it beyond 4GB.
My preference would be to leave 32-bit build as-is.

In ye olden days this base address was a way to maximise start-up performance by ensuring that a set of DLLs could load without needing their base address adjusted at runtime.
